icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -636,6 +637,9 @@ class StorageSettings:
             Store chunk objects using this object store storage class.
             Currently not supported in GCS.
             Defaults to storage_class.
+
+        minimum_size_for_multipart_upload: int | None
+            Use object store's multipart upload for objects larger than this size in bytes.
```

Comment:
Can we add the default to the docstring?

Reply:
good call! done